chart/k8gb/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: k8gb
 description: A Helm chart for Kubernetes Global Balancer
 type: application
-version: v0.7.7
-appVersion: v0.7.7
+version: v0.7.8
```

Comment:
Are we doing release?

Reply:
@k0da not at the moment. 

Still whenever we make core controller modifications we tend to bump the version even without an official helm release.

It will make git-based helm installations consistent and straightforward, avoiding confusing situations when the tip of the branch contains the fix meanwhile helm chart configuration points to the old controller build

Reply:
Then we should really split chart and appVersions

Reply:
@ytsarev IMO since this version bumping rule is not obvious (I can hardly remember it myself :) ), it should be reflected in contributors documentation. 
I remember there were we pretty serious changes and refactoring made in controller code, and these were not reflected in chart version bump, so we should be consistent here.
Also, what happens if I try to checkout the project and deploy full cluster setup locally, when chart version is bumped, but release is not made, hence docker image for corresponding version is not publicly available?

Reply:
@k0da could you please elaborate why? I really fail to see why we need to add additional complexity

@somaritane it was almost the very beginning of the project, we agreed to keep it simple and keep appVersion and Version in sync until it is really necessary to deviate. Currently, it is more of tribe knowledge. Good idea to document it in general, not in scope of this PR though.
`v0.7.8` is available in the public docker hub repo but guarded against public consumption by unreleased helm chart.

Reply:
@ytsarev, was `v0.7.8` pushed to docker hub manually?

Reply:
It just leads to confusion. And git based installation would imply setting override tag eitherway. v0.7.8 doesn't reflect any git status.

Reply:
@somaritane yep, with 
```
make docker-build-multi
make docker-push-multi
```
Again, it is just a tag and does not indicate any release

@k0da deviation between appVersion and Version leads to even more confusion.

Anyway, what we are discussing here? I can exlcude chart version bump from PR making it effectively not-installable at the moment. Is that what we want?

Reply:
I'd preffer to not include chart version bump.

What do you mean by not-installable? You can always set image version.

Reply:
Meaning that the tip of the master branch will install an old version of the controller by default.

Ok, I will exclude it, no worries

Reply:
@k0da @somaritane no version bump anymore

Reply:
@ytsarev , I think we should avoid manual publishing of docker images with version tag outside of release process. There's no guaranty that the image really reflects the claimed version and git state and also it is prone to errors.

For example, I can see the following image Docker Hub https://hub.docker.com/layers/absaoss/k8gb/v0.7.8-arm64/images/sha256-4c13f5931cbdfda226f915affbecf8ce9ffe856da0cf1575401eba4a7bbcb1cd?context=explore,
which has `arm64` arch in the version, but at the same time has `linux/amd64` arch in metadata.

Reply:
then we have a bug in make targets.

Again, it was not a release tag and not available for end users. But I understand the confusion, let's keep the stable-looking versions to the release pipelines.